tests/actions/validate/test_resource.py

```diff
@@ -1199,3 +1199,13 @@ def test_validate_resource_duplicate_labels_with_sync_schema_issue_910():
             'Duplicate labels in header is not supported with "schema_sync"',
         ],
     ]
+
+
+def test_validate_invalid_resource_long_error_messages():
+    report = validate("data/datapackage.json", type="resource")
+    assert report.flatten(["code", "note"]) == [
+        [
+            "resource-error",
+            "\"{'format': 'inline',\n 'hashing': 'md5',\n 'name': 'test-tabulator',\n 'profile': 'tabular-data-resource',\n 'resources': [{'name': 'first-resource',\n                'path': 'table.xls',\n                'schema': {'fields': [{'name': 'id', 'type': 'number'},\n                                      {'name': 'name', 'type': 'string'}]}},\n               {'name': 'number-two',\n                'path': 'table-reverse.csv',\n                'schema': {'fields': [{'name': 'id', 'type': 'integer'},\n                                      {'name': 'name', 'type': 'string'}]}}],\n 'scheme': '',\n 'stats': {'bytes': 0, 'fields': 0, 'hash': '', 'rows': 0}} is not valid under any of the given schemas\" at \"\" in metadata and at \"oneOf\" in profile",
```

Comment:
Python validate call shoud not be related to the CLI so not sure about this test